.buildkite/bk.integration-fips.pipeline.yml

```diff
@@ -4,7 +4,7 @@ env:
   DOCKER_REGISTRY: "docker.elastic.co"
   ASDF_MAGE_VERSION: 1.14.0
 
-  IMAGE_UBUNTU_2404_X86_64: "platform-ingest-elastic-agent-ubuntu-2404-1744855248"
+  IMAGE_UBUNTU_2404_X86_64: "family/platform-ingest-elastic-agent-ubuntu-2404"
```

Comment:
I believe this will undo the buildkite image pinning we introduced recently (@pkoutsovasilis keep me honest here).
@michel-laterman why is this change needed ?